iron-iconset-svg.html

```diff
@@ -69,6 +69,10 @@
       }
 
     },
+    
+    attached: function() {
+      this.style.display = 'none';
```

Comment:
:+1: I think you're right that we might end up creating an unnecessary `shadowRoot`. Alternatively you could also set `hostAttributes` to include `hidden` by default. This may be more expensive overall though..